srcpkgs/tali/template

```diff
@@ -9,5 +9,6 @@ short_desc="GNOME sort of poker game with dice"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="GPL-2.0-or-later"
 homepage="https://wiki.gnome.org/Apps/Tali"
+changelog="${GNOME_SITE}/tali/${version%.*}/tali-${version}.news"
```

Comment:
hmm, I personally prefer a changelog from gitlab because I can easily just edit the URL to get to the major versions changelog or get to the repository.

Reply:
I agree, sorry I forgot to look on gitlab